.github/workflows/ci.yaml

```
@@ -20,12 +19,38 @@ jobs:
-          - windows-latest
-    name: Build and Test 🔬 - ${{ matrix.os }}
-    runs-on: ${{ matrix.os }}
+        include:
+            # Linux x64 (default)
+            - name: ubuntu-x64
+              runs-on: ubuntu-latest
+              arch: x64
+  
+            # Linux arm64 - use GitHub hosted ARM runner (native) when available
+            - name: ubuntu-arm64
+              runs-on: ubuntu-latest-arm
```

actionlint (1.7.9)

label "ubuntu-latest-arm" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xl", "macos-latest-xlarge", "macos-latest-large", "macos-26-xlarge", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xl", "macos-14-xlarge", "macos-14-large", "macos-14", "macos-13-xl", "macos-13-xlarge", "macos-13-large", "macos-13", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)